CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 # Changelog
-# Version 3.38.0 (2023-xx-xx)
+# Version 3.38.0 (2023-02-15)
 
-### Added
+## Added
 * All imports are available via `import labelbox as lb` and `import labelbox.types as lb_types`.
+* Attachment_name support to create_attachment()
 
 ## Changed
 * `LabelImport.create_from_objects()`, `MALPredictionImport.create_from_objects()`, `MEAPredictionImport.create_from_objects()`, `Project.upload_annotations()`, `ModelRun.add_predictions()` now support Python Types for annotations.
+* Ontology (instructions --> name)
```

Comment:
Let's add this part into `## Notebooks` section, since the code change was made in 3.37.0, and only the notebooks changed between 3.38.0
`* Updated notebooks to use Classification.name instead of Classification.instructions`